mmdet3d/apis/inference.py

```diff
@@ -236,18 +267,75 @@ def show_result_meshlab(data,
         show=show,
         snapshot=snapshot)
 
-    if 'img' not in data.keys():
-        return out_dir, file_name
+    return file_name
+
+
+def show_seg_result_meshlab(data,
+                            result,
+                            out_dir,
+                            palette,
+                            show=False,
+                            snapshot=False):
+    """Show 3D segmentation result by meshlab."""
+    points = data['points'][0][0].cpu().numpy()
+    pts_filename = data['img_metas'][0][0]['pts_filename']
+    file_name = osp.split(pts_filename)[-1].split('.')[0]
+
+    pred_seg = result[0]['semantic_mask'].numpy()
+
+    if palette is None:
+        # generate random color map
+        max_idx = pred_seg.max()
+        palette = np.random.randint(0, 256, size=(max_idx + 1, 3))
+    palette = np.array(palette).astype(np.int)
+
+    show_seg_result(
+        points,
+        None,
+        pred_seg,
+        out_dir,
+        file_name,
+        palette=palette,
+        show=show,
+        snapshot=snapshot)
+
+    return file_name
+
 
-    # multi-modality visualization
-    # project 3D bbox to 2D image plane
+def show_proj_bbox_img_result_meshlab(data,
```

Comment:
The name is a little long? May be changed to something like `show_proj_det_result_meshlab` or `show_img_det_result_meshlab` (similar to the previous `show_det_result_meshlab`)

Reply:
Fix.